docs/TypeScript.md

```diff
@@ -0,0 +1,116 @@
+<h1 align="center">Fastify</h1>
+
+## TypeScript
+Fastify is shipped with the a typings file so it should "just work" when your used in a TypeScript application.
```

Comment:
`should "just work" when your used in a TypeScript application.` => `should "just work" when used in a TypeScript application.`